src/components/pages/Account.tsx

```diff
@@ -20,32 +28,69 @@ type DispatchProps = {
 
 type StateProps = {
   loggedInMember: Member;
+  trustedForRecovery: Member;
+  votingFor: Member;
+};
+
+type Props = DispatchProps & OwnProps & StateProps;
```

Comment:
Not sure how important this is, but I think we usually do AccountProps / AccountState?

Reply:
Grep shows 14 instances of just `Props`, can we make standard be that if file has multiple `Props` then disambiguate all of them, but if just one okay to leave as `Props`?

Reply:
@osdiab  since I don't really know what normal style is!